scrapingbee_fetch_news/program.py

```diff
@@ -6,15 +6,19 @@
 from autokitteh.slack import slack_client
 
 SB_API_KEY = os.getenv("SB_API_KEY")
-NEWS_URL = os.getenv("NEWS_URL", "https://www.theguardian.com/international/rss")
+NEWS_WEBSITE_URL = os.getenv(
+    "NEWS_WEBSITE_URL", "https://www.theguardian.com/international/rss"
+)
 SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "#news")
 
 client = ScrapingBeeClient(api_key=SB_API_KEY)
 slack = slack_client("slack_conn")
 
 
 def on_app_mention(event):
-    feed = client.get(NEWS_URL, params={"render_js": False, "block_resources": True})
+    feed = client.get(
+        NEWS_WEBSITE_URL, params={"render_js": False, "block_resources": True}
+    )
     if feed.status_code != 200:
         print("Feed fetch failed:", feed.status_code, feed.text[:300])
         return
```

Comment:
When the RSS feed fetch fails, the function returns silently without notifying the user. Since they mentioned the app in Slack, they should receive feedback about the failure:

```suggestion
if feed.status_code != 200:
      slack.chat_postMessage(
          channel=SLACK_CHANNEL,
          text=f"Failed to fetch news feed: HTTP {feed.status_code}"
      )
      print("Feed fetch failed:", feed.status_code, feed.text[:300])
      return
```